### DISCUSSION TITLE
Add a "does not contain" option in the search box

### DISCUSSION THREAD

**Discussion by flaviolivolsi** [plausible/analytics#2681] [Anything Else]
Currently there are 3 options in the filters: is, is not and contains. It would be great to have "does not contain" as well.

**Comment by metmarkosaric** [comment]
thanks @flaviolivolsi! i think this might work already for pages. you could try to use the "is not" option with wildcard filter in this format `**keyword**`. that should list all pages that don't contain that specific keyword. we have a bit more on these wildcard filters here: https://plausible.io/docs/filters-segments#group-your-pages

**Reply by flaviolivolsi** [reply]
Cool! Thank you @metmarkosaric :)
